packages/terra-functional-testing/package.json

```diff
@@ -37,6 +37,8 @@
     "@wdio/local-runner": "^6.1.4",
     "@wdio/mocha-framework": "^6.1.0",
     "@wdio/spec-reporter": "^6.0.16",
+    "@wdio/sync": "^6.1.14",
```

Comment:
did this dep needed to be added?

Reply:
Yes, the existence of the package is the flag for running wdio synchronously.

https://webdriver.io/docs/sync-vs-async.html

It's the equivalent to this:

https://github.com/cerner/terra-toolkit-boneyard/blob/master/config/wdio/wdio.conf.js#L71